lib/active_remote/cached.rb

```diff
@@ -24,6 +24,10 @@ def self.cache(cache_provider = nil)
       @cache_provider
     end
 
+    def self.cache_options
+      ::ActiveRemote::Cached.default_options.slice(:expires_in, :namespace, :race_condition_ttl)
```

Comment:
Probably a good thing to do here. This could break things if folks have set other params for their custom cachers. I don't think this will break anything for MX, though. The other callout would be that the keys may or may not be keywords. I think strings would also work for the older version?